src/main/default/lwc/flowProgressIndicator/__tests__/flowProgressIndicator.test.js

```diff
@@ -0,0 +1,187 @@
+  virtual: true
+});
+jest.mock(
+  '@salesforce/label/c.cookbook_DescribeProblem',
+  () => ({ default: 'Describe problem' }),
+  {
+    virtual: true
+  }
+);
+jest.mock(
+  '@salesforce/label/c.cookbook_ProvideDetails',
+  () => ({ default: 'Provide details' }),
+  {
+    virtual: true
+  }
+);
+jest.mock(
+  '@salesforce/label/c.cookbook_ReviewHelp',
+  () => ({ default: 'Review help' }),
+  {
+    virtual: true
+  }
+);
+
+const DEFAULT_STEPS = [
+  {
+    label: 'Log in',
+    value: 'log in'
+  },
+  {
+    label: 'Describe problem',
+    value: 'describe problem'
+  },
+  {
+    label: 'Provide details',
+    value: 'provide details'
+  },
+  {
+    label: 'Review help',
+    value: 'review help'
+  }
+];
+
+const CUSTOM_STEPS = [
+  {
+    label: 'First step',
+    value: 'first step'
+  },
+  {
+    label: 'Second step',
+    value: 'second step'
+  }
+];
+
+const assertProgressStepEquals = (actual, expected) => {
+  expect(actual).not.toBeNull();
+  expect(actual.label).toBe(expected.label);
+  expect(actual.value).toBe(expected.value);
+};
+
+describe('c-flow-progress-indicator', () => {
+  afterEach(() => {
+    // The jsdom instance is shared across test cases in a single file so reset the DOM
+    while (document.body.firstChild) {
+      document.body.removeChild(document.body.firstChild);
+    }
+  });
+
+  it('should display the progress indicator', async () => {
+    const element = createTestComponent();
+
+    await allPromisesResolution();
+    const progressIndicatorNode = element.shadowRoot.querySelector(
+      'lightning-progress-indicator'
+    );
+
+    expect(progressIndicatorNode).not.toBeNull();
+    expect(progressIndicatorNode.currentStep).toBe(undefined);
+    await expect(element).toBeAccessible();
+  });
+
+  it('should display the correct default progress indicator steps', async () => {
+    const element = createTestComponent();
+
+    await allPromisesResolution();
+    const progressIndicatorNode = element.shadowRoot.querySelector(
+      'lightning-progress-indicator'
+    );
+    const stepNodes = element.shadowRoot.querySelectorAll(
+      'lightning-progress-step'
+    );
+
+    expect(progressIndicatorNode).not.toBeNull();
+    expect(stepNodes.length).toBe(DEFAULT_STEPS.length);
+    stepNodes.forEach((step, idx) =>
+      assertProgressStepEquals(step, DEFAULT_STEPS[idx])
+    );
+    await expect(element).toBeAccessible();
+  });
+
+  it('should display the correct progress indicator steps', async () => {
+    const element = createTestComponent();
+    element.steps = CUSTOM_STEPS;
+
+    await allPromisesResolution();
+    const progressIndicatorNode = element.shadowRoot.querySelector(
+      'lightning-progress-indicator'
+    );
+    const stepNodes = element.shadowRoot.querySelectorAll(
+      'lightning-progress-step'
+    );
+
+    expect(progressIndicatorNode).not.toBeNull();
+    expect(stepNodes.length).toBe(CUSTOM_STEPS.length);
+    stepNodes.forEach((step, idx) =>
```

Comment:
Same as above here, I would prefer you run your forEach on the CUSTOM_STEPS instead of on the nodes itself.